tests/FSharp.Compiler.ComponentTests/TypeChecks/Graph/CompilationFromCmdlineArgsTests.fs

```diff
@@ -5,19 +5,19 @@ module CompilationFromCmdlineArgsTests =
     open System
     open System.IO
     open FSharp.Compiler.CodeAnalysis
-    open NUnit.Framework
+    open Xunit
     open CompilationTests
 
     // Point to a generated args.txt file.
     // Use scrape.fsx to generate an args.txt from a binary log file.
     // The path needs to be absolute.
-    let localProjects: string list =
+    let localProjects =
         [
             @"C:\Projects\fantomas\src\Fantomas.Core\Fantomas.Core.args.txt"
             @"C:\Projects\FsAutoComplete\src\FsAutoComplete\FsAutoComplete.args.txt"
             @"C:\Projects\fsharp\src\Compiler\FSharp.Compiler.Service.args.txt"
             @"C:\Projects\fsharp\tests\FSharp.Compiler.ComponentTests\FSharp.Compiler.ComponentTests.args.txt"
-        ]
+        ] |> Seq.map (fun p -> [| box p |])
```

Comment:
This is Xunit specific, `MemberData` must be `obj array list` or `obj array array`. We do this in all similar places, e.g. [here](https://github.com/dotnet/fsharp/blob/398d16d95c14c96f5ff7173b5fa529fa879cbee6/tests/FSharp.Compiler.ComponentTests/Conformance/BasicGrammarElements/StaticLet/StaticLetInUnionsAndRecords.fs#L13).